tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,11 @@
+
 const App = () => {
-	return <div>A tic-tac-toe game</div>;
+	return (
```

Comment:
Each of this app's components is rendering a logical child component. Using modules generally results in code that is easier to read and maintain. 💯